### ISSUE TITLE
systray icon in 4k display (non-HDPI panel) (plasma5)

### ISSUE THREAD

**Issue by sl1pkn07** [crmarsh/discord-linux-bugs#172]
a pic:

![screenshot_20170119_172629](https://cloud.githubusercontent.com/assets/462213/22115237/844067dc-de6c-11e6-89db-7907272a752f.png)

implement SVG instead of fixed-size icons i think fix the issue

implement StatusNotifierItem (http://www.notmart.org/misc/statusnotifieritem/) help in plasma5 environment

greetings

**Comment by OvermindDL1** [comment]
Oh, wow, just noticed this too, non-svg icons are still used?  o.O

**Comment by sl1pkn07** [comment]
i not sure, because https://github.com/crmarsh/discord-linux-bugs/issues/168 XD, but sound yes

**Comment by Serkan-devel** [comment]
I had that same issue when I was on kde.
Doesn't seem stay the same on lxqt

**Comment by crmarsh** [comment]
I feel like I tried to use a svg and it didn't work, so I switched to png, but maybe that was somewhere else. Oh, [here's the doc](https://github.com/electron/electron/blob/master/docs/api/native-image.md) -- it says png and jpeg are supported. They have a thing where you provide multiple images for different scales and it picks the right one, though, so I probably should at least use that.

**Comment by Hacksaurus** [comment]
Sorry for commenting on a closed topic, but how would I adjust my icon to make it not so blurry?
I am running Arch, with XFCE where the blurriness happens, but when using Budgie there is not blurriness...

**Comment by skwerlman** [comment]
@Hacksaurus this issue is still open, so no worries; afaik there is currently not a workaround for this

**Comment by Hacksaurus** [comment]
Ah I c... Hmmm I wish there was an something like a secret settings file for Linux or something to solve this... But thats not a very good workaround...

**Comment by sl1pkn07** [comment]
I have tried replaze the systray icons in '/home/$HOME/.config/discordcanary/':

![linux-tray](https://user-images.githubusercontent.com/462213/29248712-31615c92-801f-11e7-88a1-ee8175484a72.png)
![linux-tray-unread](https://user-images.githubusercontent.com/462213/29248715-3468d8ca-801f-11e7-883b-30d765acd838.png)

by this others:

![linux-tray](https://user-images.githubusercontent.com/462213/29248732-ab74075a-801f-11e7-89ba-43b624b8f74a.png)
![linux-tray-unread](https://user-images.githubusercontent.com/462213/29248733-b0505634-801f-11e7-906b-4c15b3d572e4.png)


and then restart discord. but always replace the newers with the olders.... this is sad :(

**Comment by Hacksaurus** [comment]
Now that this thread has been revived, I actually noticed that they have fixed this already:

https://lh3.googleusercontent.com/-zOYFm-3-9Wk/WZBwLL7ZXrI/AAAAAAAANVQ/awloESlA49oK_-8EocPVxa8eoYyp_YydQCL0BGAYYCw/h35/Screenshot_2017-08-13_08-27-48.png

As you can see there is really any blur to the systray icon anymore.

**Comment by sl1pkn07** [comment]
If you mean with 0.0.24, still show blurry icon in 4k displays

**Comment by Hacksaurus** [comment]
Oh, this is for Canary, it works perfectly fine with the normal Discord

**Comment by sl1pkn07** [comment]
tested with normal discord (0.0.2). still ship 24x24px systray icons

**Comment by Hacksaurus** [comment]
Huh, this might be an issue with your bar... I am using polybar, and there is an option to override the current size of the icons in the systemtray, and I currently set them at 32, I believe...

**Comment by sl1pkn07** [comment]
nope. because other systray icons show OK (in SVG format, breeze-dark icon theme). also telegram i make the same workground as exposed above (replace with other icons with more resolution, 256x256px in my case) and show nicely

also i uninstall libappindicator-gtk{2,3} and do nothig (same blurry icons). 

the problem is the 24x24px images. but can't test it because always is remplaced by older ones when run/reload discord

**Comment by Hacksaurus** [comment]
Then I don't really know whats the problem... I feel like this is an issue particular to you...
Sorry, but I don't really have any other solutions for ya...

**Comment by sl1pkn07** [comment]
https://bugs.chromium.org/p/chromium/issues/detail?id=382487

if any chromium dev can fix this... then ported to electron (based on chromium)....
